### ISSUE TITLE
Upgrade sonar-cxx-plugin version

### ISSUE THREAD

**Issue by gbonnefille** [lequal/docker-cat#26]
Running this version on C++ project I encounter an error:
```
ERROR: Error during SonarQube Scanner execution
ERROR: Metric 'files' should not be computed by a Sensor
```
Looking around it seems to be an incompatibility between SonarQube version an plugins as SonarQube depreciated a 'files' metric. Example: https://github.com/mwz/sonar-scala/issues/181

Looking at https://github.com/SonarOpenCommunity/sonar-cxx/wiki/SonarQube-compatibility-matrix we need 1.3.1 at least for 7.9 compatibility.

**Comment by Sancretor** [comment]
Hi @gbonnefille !

Thanks for this new contribution, we appreciate it !
We'll take some time to do tests, then give you our feedback... but it seems ok for me.

**Comment by Sancretor** [comment]
Hi @gbonnefille !

We reproduced your anomaly, and your fix resolved it.
Your PR will be merged, and the 2.0.1 patch will be released then.

Thanks again for your contribution !
